### PR TITLE
Add missing guard for user id on audit log entry

### LINKED ISSUE #1057: No verification email sent when admin marks a user email as unverified
### Current Behavior

If an admin changes the 'Email Verified' flag for a user, no email is sent to the user to prompt them to reverify.

When that user next comes to the platform they are asked to click the link in the email... an email they haven't been sent. We do provide a button to resend the email, but the current flow will send the user off looking for an email that wasn't sent.

### Expected Behavior

I *think* we should send the email automatically when admin resets the email verified flag.

### Steps To Reproduce

- As admin, edit a user via Admin Settings to set Email Verified to false
- As that user, observe that no email was received

### Environment

- FlowForge version:
- Node.js version:
- npm version:
- Platform/OS:
- Browser:

### PR DESCRIPTION
**Pull request by Steve-Mcl**
addresses https://github.com/flowforge/flowforge/issues/1057#issuecomment-1290304861

fixes #1057

### CODE REVIEW

**Review by hardillb — COMMENTED** [review]
Should line 381 be the same?

**Comment by Steve-Mcl** [comment]
> Should line 381 be the same?

Resolved in [031ffb2](https://github.com/flowforge/flowforge/pull/1132/commits/031ffb288db38d0e38a3959dd1b7e5b09b2a78c2)

File: forge/routes/auth/index.js [hunk 1]
```diff
@@ -399,14 +399,14 @@ module.exports = fp(async function (app, opts, done) {
...
-            await userLog(request.session.User.id, 'verify.verify-token', {
+            await userLog(request.session?.User?.id || verifiedUser.id, 'verify.verify-token', {
                 user: {
                     username: verifiedUser.username,
                     name: verifiedUser.name,
                     email: verifiedUser.email,
                     admin: !!verifiedUser.isAdmin
                 }
-            })
+            }, verifiedUser?.id)
```
Comment (hardillb) [inline forge/routes/auth/index.js:409]: Not sure we need this (especially as we've used it unguarded in the 2 other changed lines
```suggestion
            }, verifiedUser.id)
```

Comment (Steve-Mcl) [reply forge/routes/auth/index.js:409]: yup - you are right

**Review by hardillb — APPROVED** [review]

**Pull request merged**